tests/tap_hold_configurations/chordal_hold/permissive_hold/test_one_shot_keys.cpp

```diff
@@ -0,0 +1,148 @@
+/* Copyright 2021 Stefan Kerkmann
```

Comment:
I would assume you need to add yourself here and the current year, I guess this and the other files just got that  header copied?

Reply:
Thanks for the catch! Done.